shell.nix

```diff
@@ -57,12 +57,14 @@ let
       git python27 curl jq
       nodePackages.node-gyp nodePackages.node-pre-gyp
       gnumake
-      chromedriver
+      (if system == "aarch64-darwin"
+        then (localLib.iohkNix.getPkgs { system = "x86_64-darwin"; inherit config; }).chromedriver
```

Comment:
In https://github.com/NixOS/nixpkgs/commit/6e0a2c1c5422706447a34f71c48f6c37f0a6542d#diff-e846459ce0553983db1a53479d09dc25fe71fab9f13db893ab85f35d15b6172d nix added support for aarch64 chromedriver, removing the need for this

Reply:
Thank you @Diatrus!